pages/franchise_page/[franchise].md

```diff
@@ -128,15 +127,60 @@ order by slot asc
 > League Selection
 <Dropdown data={player_info} name=League value=league />
 
+```sql staff_members1
```

Comment:
The name of this query and the SQL it contains should be much more specific (this is good to be mindful of everywhere) because the more we're pulling via SQL the worse the performance will be.  We should just be pulling the name of captains (and GMs if that's who you're wanting to have be in place of not having a captain) as opposed to pulling a ton of stuff that isn't being used if this is only feeding a single Big Value.  Alternatively, the best solution would be to modify your above staff_members query so that you don't need a separate query and both visuals can run off of one query.

Additionally, putting the GM in as the captain didn't work for my AL team.  It put Pikachew_RL as the captain since Mr. Clean was banned a week ago.  However, Pika isn't FM, GM, AGM, or a captain, so I think you might need to figure out what went wrong here.